components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx

```diff
@@ -122,9 +122,10 @@ export const NetworkFilterSelector = ({ networkListSubset }: Props) => {
                   {sortedNetworks.filter((n) =>
```

Comment:
See line #120 - what happens if `isTestNetworksEnabled` is false?

Reply:
If `isTestNetworksEnabled` is `false` we do not show the `subMenu`